diff --git a/node_modules/react-native-agora/android/build.gradle b/node_modules/react-native-agora/android/build.gradle
index e097841..c8f47ff 100644
--- a/node_modules/react-native-agora/android/build.gradle
+++ b/node_modules/react-native-agora/android/build.gradle
@@ -166,7 +166,6 @@ dependencies {
   implementation fileTree(include: ['*.jar', '*.aar'], dir: 'libs')
   /// dependencies start
   api 'io.agora.rtc:agora-special-full:4.5.3.70'
-  implementation 'io.agora.rtc:full-screen-sharing-special:4.5.3.70'
   api 'io.agora.rtc:iris-rtc:4.5.3-build.1'
   /// dependencies end
 }
